ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -769,3 +769,10 @@ FAN_SS_MSM:
   implements:
   - FAN
   - MSM
+
+FAN_HLC:
```

Comment:
Please include `SS` here as well to capture the general fan on/off status.

Reply:
Done.